x-pack/plugins/elastic_assistant/server/lib/langchain/content_loaders/esql_loader.ts

```diff
@@ -76,11 +76,10 @@ export const loadESQL = async (esStore: ElasticsearchStore, logger: Logger): Pro
       `Loading ${docsWithMetadata.length} ES|QL docs, ${languageDocsWithMetadata.length} language docs, and ${requiredExampleQueries.length} example queries into the Knowledge Base`
     );
 
-    const response = await esStore.addDocuments([
-      ...docsWithMetadata,
-      ...languageDocsWithMetadata,
-      ...requiredExampleQueries,
-    ]);
+    const response = await kbDataClient.addKnowledgeBaseDocuments({
+      documents: [...docsWithMetadata, ...languageDocsWithMetadata, ...requiredExampleQueries],
+      global: true,
+    });
```

Comment:
`esStore` was just passing this along to this `kbDataClient` call anyway, so was able to swap usages and remove what was one of the last `esStore` dependencies.